net/openvswitch/Makefile

```diff
@@ -301,10 +301,10 @@ CONFIGURE_ARGS+= \
 
 CONFIGURE_VARS += \
 	ovs_cv_flake8=no \
-	ovs_cv_python3=no \
+	ovs_cv_python=$(PYTHON3) \
+	ovs_cv_python_host=$(HOST_PYTHON3_BIN) \
 	ovs_cv_sphinx=no \
-	ovs_cv_python=$(PYTHON) \
-	ovs_cv_python_host=$(HOST_PYTHON_BIN) \
+	ovs_cv_python2=no \
```

Comment:
if doing this, does it make sense to have both python & python3 variants ?
i forget about the internals of the OVS build here;
it could be that packaging for both Python & Python3 should work fine here

Reply:
Yes, the ovs python code was written in a way compatible with both python2 and python3.

Here we are just instructing the build system to use python3 for building and as shebang pythin utils on target machine.